core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/ScheduleGenerator.scala

```diff
@@ -153,11 +157,11 @@ abstract class ScheduleGenerator(
       .removeLink(physicalLink)
 
     // create the uri of the materialization storage
-    val storageUri = VFSURIFactory.createMaterializedResultURI(
+    val storageUri = VFSURIFactory.createResultURI(
```

Comment:
Please avoid using `result`. use `createMaterializationURI` instead.

Reply:
`createMaterializedResultURI` is removed. `createResultURI` is the existing storage-layer API method name used by sink operators. If we want to change this name I suggest we do a separate PR because we are also using "result" throughout the storage layer, like `operator-port-result`. Also "materialization" is not necessarily a better name since it has specific meanings in the database context (materialized query intermediate results as opposed to user-desired outputs)